tools/partutil/helpers.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package partutil
+
+import (
+	"errors"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+)
+
+// Check checks whether err is returned and print error message.
+func Check(e error, msg string) bool {
+	if e != nil {
+		log.Printf("ERROR!!!!\n %s\n %s\n\n", msg, e.Error())
+		return true
+	}
+	return false
+}
+
+// ExecCmdToStdout runs a command and output to stdout.
+func ExecCmdToStdout(cmdLine string) error {
+	cmd := exec.Command("/bin/bash", "-c", cmdLine)
+	cmd.Stdout = os.Stdout
+	return cmd.Run()
+}
+
+// ConvertSizeToBytes converts a size string to int unit: bytes.
+// It takes a string of number with no unit (sectors), unit B, unit K, unit M, or unit G.
+func ConvertSizeToBytes(size string) (int, error) {
+	const B = 1
+	const K = 1024
+	const M = K * 1024
+	const G = M * 1024
+	const SEC = 512
+
+	var err error
```

Comment:
Any specific reason to create it here?

Reply:
because I use `res` to store the result. If `err` is declared like `if res, err := strconv.Atoi(size[0 : l-1]);err!=nil{...}`, `res` will be taken as a local variable in the block, and its change will not be shown on the function scope variable `res`